rpc/ethereum/types/utils.go

```diff
@@ -282,6 +282,30 @@ func FindTxAttributes(events []abci.Event, txHash string) (int, map[string]strin
 	return -1, nil
 }
 
+// FindTxAttributesByIndex search the msg in tx events by txIndex
+// returns the msgIndex, returns -1 if not found.
+func FindTxAttributesByIndex(events []abci.Event, txIndex uint64) int {
+	strIndex := []byte(strconv.FormatUint(txIndex, 10))
+	msgIndex := -1
+	for _, event := range events {
+		if event.Type != evmtypes.EventTypeEthereumTx {
+			continue
+		}
+
+		msgIndex++
+
+		value := FindAttribute(event.Attributes, []byte(evmtypes.AttributeKeyTxIndex))
```

Comment:
we should move `[]byte(evmtypes.AttributeKeyTxIndex)` out of the `for` loop to avoid unnecessary conversions

Reply:
done